### ISSUE TITLE
Shadowfinder 2.0

### ISSUE THREAD

**Issue by GaborFriesen** [bellingcat/ShadowFinder#23]
Now we use object height/shadow length at a known time to identify a list of potential locations.
If you have a shadow at 2 different times, with length A and length B, then the proportional change from A to B (or B to A) for that specific change from time A to time B corresponds to a specific list of potential locations. The general theory is the same, just the inputs are different. 
Having a section that would allow for an input like this could increase the potential use cases for the shadow finder.

In other words. Right now the inputs are:
Object height + Shadow length + known time = locations.
I propose adding a new section with the inputs
Shadow length 1 + Time 1 + Shadow length 2 + Time 2 = locations
